addons/transifex/models/ir_translation.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, fields
+
+
+class IrTranslation(models.Model):
+
+    _inherit = 'ir.translation'
+
+
+    def _get_transifex_url(self):
+        """ Construct transifex URL based on the module on configuration """
+        # e.g. 'https://www.transifex.com/odoo/odoo-10'
+        base_url = self.env['ir.config_parameter'].sudo().get_param('transifex.project_url')
+
+        if not base_url:
+            self.transifex_url = False
+        else:
+            base_url = base_url.rstrip('/')
+
+            # will probably be the same for all terms, avoid multiple searches
+            languages = {}
+            for translation in self:
+                if not translation.module or not translation.source or translation.lang == 'en_US':
+                    # custom or source term
+                    continue
+
+                lang_code = languages.get(translation.lang)
+                if not lang_code:
+                    lang_code = self.env['res.lang'].with_context(active_test=False).search(
+                        [('code', '=', translation.lang)],
+                        limit=1).iso_code
+                    languages[translation.lang] = lang_code
+                if not lang_code:
+                    continue
+
+                translation.transifex_url = "%(url)s/translate/#%(lang)s/%(module)s/42?q=%(src)s" % {
+                    'url': base_url,
+                    'lang': lang_code,
+                    'module': translation.module,
+                    'src': translation.source[:15],
+                }
+
+    transifex_url = fields.Char("Transifex URL", compute=_get_transifex_url)
```

Comment:
Field should be first, and compute value should be a string (name of the method) so it can be overridden easily.